app/_includes/docs-sidebar-item.html

```diff
@@ -6,7 +6,7 @@
   {% for item in include.items %}
   {% if item.items %}
   {% assign id = include.id | append: '-' | append: forloop.index %}
-  {% include docs-sidebar-item.html id=id docs_url=include.docs_url items=item.items title=item.text url=item.url absolute_url=item.absolute_url active=item_active %}
+  {% include docs-sidebar-item.html id=id url=include.url docs_url=include.docs_url items=item.items title=item.text url=item.url absolute_url=item.absolute_url active=item_active %}
```

Comment:
This statement is declaring `url` twice: `url=include.url` and `url=item.url`. This is causing multiple nav entries to be flagged as active: https://deploy-preview-3080--kongdocs.netlify.app/enterprise/2.5.x/support-policy/